webapp/src/utils/github_utils.js

```diff
@@ -0,0 +1,7 @@
+export function isUrlCanPreview(url) {
+    if (url.includes('github.com/') && url.split('github.com/')[1]) {
+        const [owner, repo, type, number] = url.split('github.com/')[1].split('/');
+        return !(!owner | !repo | !type | !number);
```

Comment:
Bit-wise or's 🤔  I see this is also being used in the tooltip code

We can do this to make it a bit more readable

```suggestion
        return Boolean(owner && repo && type && number);
```